zonos/model.py

```
@@ -190,104 +293,109 @@ def _prefill(
+        return can_use_cudagraphs_static(self.device, str(self.backbone.__class__))
 
     @torch.inference_mode()
     def generate(
         self,
         prefix_conditioning: torch.Tensor,  # [bsz, cond_seq_len, d_model]
-        audio_prefix_codes: torch.Tensor | None = None,  # [bsz, 9, prefix_audio_seq_len]
+        audio_prefix_codes: torch.Tensor = None,  # [bsz, 9, prefix_audio_seq_len]
         max_new_tokens: int = 86 * 30,
         cfg_scale: float = 2.0,
         batch_size: int = 1,
         sampling_params: dict = dict(min_p=0.1),
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)